pkg/config/teyps_test.go

```diff
@@ -0,0 +1,199 @@
+package config
```

Comment:
I'm still in the process of looking at the whole thing, but it seems that this file name is typo.